### ISSUE TITLE
Add "New regex" button

### ISSUE THREAD

**Issue by OSjerick** [firasdib/Regex101#712]
Can you add a "New regex" button? If you save some regex, the current page is linked to it, so if you want to start another regex without making changes to that regex, you have to reopen the app.

**Comment by TWiStErRob** [comment]
Have you tried clicking the website logo?

On most websites if you click the logo it goes to the home page, which in this case should be a reset editor.

**Comment by TWiStErRob** [comment]
You cane even middle click it to open a new editor in a separate tab.

**Comment by OSjerick** [comment]
Of course I have, I'm not such ignorant. ;) I just ask for a button to make things easier and intuitive.

**Comment by Zarthus** [comment]
I think adding such a button might confuse users who don't want a clean slate. I agree it'd be easier, but all ways of implementing this I can think of involve misguiding a portion of users as well. Would need some serious thought :)

**Comment by TWiStErRob** [comment]
@Zarthus if the user doesn't want a clean slate why would they press a button called "new regex"?

If the top section "Save & share" was renamed "Regex" then "Save, Update, Fork, Share, Clear/new" are possible links there.

**Comment by Zarthus** [comment]
[If users didn't want their regex deleted why do they share the deletion link?](https://github.com/firasdib/Regex101/issues/714)

tl;dr: Users are stupid.

**Comment by OSjerick** [comment]
@TWiStErRob I think it's a good idea.

**Comment by firasdib** [comment]
I think the behavior of most apps seem to be that you just refresh the page or open a new one, to create a clean slate.
